apps/web/src/components/utils/SuccessOrError.tsx

```diff
@@ -0,0 +1,19 @@
+import { showNotification } from '@mantine/notifications';
+import { Check } from '../../design-system/icons/general/Check';
+import { ErrorIcon } from '../../design-system/icons';
+
+export function show(state: 'success' | 'error', message: string) {
+  showNotification({
+    message,
+    autoClose: false,
```

Comment:
Don't forget to remove this autoClose before the final merge :) This is good for the debugging part